iwyu_include_picker.cc

```diff
@@ -1589,9 +1611,17 @@ IncludeVisibility IncludePicker::ParseVisibility(
 }
 
 IncludeVisibility IncludePicker::GetVisibility(
-    const string& quoted_include) const {
+    const MappedInclude& include) const {
+  const IncludeVisibility* include_visibility =
+    FindInMap(&include_visibility_map_, include.quoted_include);
+  if (include_visibility) {
+    return *include_visibility;
+  }
+  if (include.path.empty()) {
```

Comment:
Is this necessary? I'm guessing we won't have the empty path in the map?

Reply:
Indeed, tests seem to pass without it.